modules/tinymce/src/core/test/ts/browser/dom/DomUtilsTest.ts

```diff
@@ -50,7 +54,7 @@ describe('browser.tinymce.core.dom.DOMUtilsTest', () => {
     // equal(dom.getAttrib('test2', 'style'), Env.ue && !window.getSelection ?
     // 'border: #00ff00 1px solid;' : 'border: 1px solid #00ff00;'); // IE has a separate output
 
-    dom.get('test').innerHTML = '<span id="test2" style="background-image: url(http://www.site.com/test.gif);"></span>';
+    (dom.get('test') as HTMLElement).innerHTML = '<span id="test2" style="background-image: url(http://www.site.com/test.gif);"></span>';
```

Comment:
Missed one?
```suggestion
    getTestElement().innerHTML = '<span id="test2" style="background-image: url(http://www.site.com/test.gif);"></span>';
```

Reply:
Oops, not sure how I missed that 😅 thanks for catching that!

Reply:
Oh wait, no this one is different as it's using `dom` not `DOM` so this wouldn't be the same conversion. While it should still work, I didn't want to convert this one in case the looks do end up diverging somehow.